specification/iotsecurity/resource-manager/readme.md

```diff
@@ -40,6 +40,10 @@ directive:
     from: onPremiseSensors.json
     where: $.definitions.OnPremiseSensor
     reason: The resource is managed in a subscription level (instead of inside a resource group)
+  - suppress: PageableOperation
```

Comment:
No need
You can (and should) define the pageable property, even if we don't plan to support it right now (yes, this is different from how some of our existing resources were originally modelled).
If one day we would actually need to page the list for some reason, we won't have to make a breaking change to the swagger

Reply:
Removed